gui/src/app/components/deployment-flow/cloud/cloud.js

```diff
@@ -89,7 +89,11 @@ class DeploymentCloud extends React.Component {
         <li
           className={classes}
           key={cloud.name}
-          onClick={this.props.setCloud.bind(null, cloud)}
+          onClick={() => {
+            this.props.setCloud(cloud);
+            this.props.analytics.addCategory('Cloud').sendEvent(
+              this.props.analytics.UPDATE, {label: `cloud: ${cloud}`});
```

Comment:
It might be clearer when we analyse the events of the category is called "Set cloud" and the label is just the cloud